Source/Scene/InvertClassification.js

```diff
@@ -167,20 +173,29 @@ const opaqueFS =
   "#endif\n" +
   "}\n";
 
-InvertClassification.prototype.update = function (context) {
+InvertClassification.prototype.update = function (
+  context,
+  numSamples,
+  globeFramebuffer
```

Comment:
Yeah, the idea of passing in the whole manager object as an argument to `update` was: 
- In the multisampling case where `previousFramebuffer` is defined (invert classification is opaque), we want to reuse both the globe framebuffer's depth-stencil renderbuffer and texture because the commands can use the unresolved depth-stencil renderbuffer as their depth test attachment. 
- If `previousFramebuffer` itself becomes a FramebufferManager object, the check for `previousFramebufferChanged` will never go through because `_previousFramebuffer` and `previousFramebuffer` will always be pointers to the same object, even when the object's framebuffers are being recreated between frames.